yt/docker/ytsaurus/credits/chyt-controller.CREDITS

```diff
@@ -1,2007 +0,0 @@
-Go (the standard library)
```

Comment:
It's used in yt/docker/strawberry, let's move it

Reply:
Fixed.